code/game/objects/map_metadata/capitol_hill.dm

```diff
@@ -392,8 +392,8 @@ var/no_loop_capitol = FALSE
 		current_win_condition = no_winner
 		win_condition.hash = 0
 		last_win_condition = win_condition.hash
-		message = "25 minutes have passed! The HVTs are now safe!"
-		world << "<font size = 4><span class = 'notice'>[message]</span></font>"
+		message = "40 minutes have passed! The HVTs are now safe!"
```

Comment:
if (processes.ticker.playtime_elapsed >= 18000) is 30 minutes, 24,000 is 40 minutes

Reply:
It ends at 39 minutes last time we played Yeltsin, hence the change.

Reply:
I'm not sure what that does.

Reply:
it checks if the time is over 18000 1/10ths of a second. 3000 = 5 minutes. 24000 = 40 minutes, 18000  30 minutes. 600 = 1 minute. Correct it to the actual timer check, it shouldnt end late unless things are contested

Reply:
Sorry, I changed the wrong thing. It was supposed to be the map_metadata for Yeltsin.

Reply:
This is resolved now.